src/webgpu/api/operation/command_buffer/compute/basic.spec.ts

```diff
@@ -0,0 +1,72 @@
+export const description = `
+Basic command buffer compute tests.
+`;
+
+import { makeTestGroup } from '../../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../../gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('memcpy').fn(async t => {
+  const data = new Uint32Array([0x01020304]);
+
+  const src = t.device.createBuffer({
+    mappedAtCreation: true,
+    size: 4,
+    usage: GPUBufferUsage.COPY_DST | GPUBufferUsage.STORAGE,
+  });
+  new Uint32Array(src.getMappedRange()).set(data);
+  src.unmap();
+
+  const dst = t.device.createBuffer({
+    size: 4,
+    usage: GPUBufferUsage.COPY_SRC | GPUBufferUsage.STORAGE,
+  });
+
+  const bgl = t.device.createBindGroupLayout({
+    entries: [
+      { binding: 0, visibility: 4, type: 'storage-buffer' },
+      { binding: 1, visibility: 4, type: 'storage-buffer' },
+    ],
+  });
+  const bg = t.device.createBindGroup({
+    entries: [
+      { binding: 0, resource: { buffer: src, offset: 0, size: 4 } },
+      { binding: 1, resource: { buffer: dst, offset: 0, size: 4 } },
+    ],
+    layout: bgl,
+  });
+
+  const pl = t.device.createPipelineLayout({ bindGroupLayouts: [bgl] });
```

Comment:
not sure if we want to rely on it in these tests, but the implicit pipeline layout can be used here